spb-land-finder/pipeline.py

```
@@ -0,0 +1,400 @@
+
+        for result in all_results:
+            if result.get('status') != 'success':
+                continue
+
+            plot_details = result.get('plot_details', {})
+            analysis = result.get('analysis', {})
+
+            # Критерии отбора
+            is_vacant = not analysis.get('is_occupied', True)
+            has_sufficient_area = plot_details.get('area', 0) >= min_area
+            is_suitable = analysis.get('suitable_for_development', False)
```

Ruff (0.14.1)

Local variable `is_suitable` is assigned to but never used

Remove assignment to unused variable `is_suitable`

(F841)